test/functional/feature_rbf.py

```diff
@@ -716,5 +720,43 @@ def test_replacement_relay_fee(self):
         tx.vout[0].nValue -= 1
         assert_raises_rpc_error(-26, "insufficient fee", self.nodes[0].sendrawtransaction, tx.serialize().hex())
 
+    def test_feerate_rule(self):
```

Comment:
Does the test at line 274 (`test_replacement_feeperkb`) not already cover this case?

Reply:
I guess this test is slightly better since it doesn't use `-acceptnonstdtxn` ? 😅 I can delete it though, I mostly wrote it for my own understanding.

Reply:
I mean I don't have a view either way on adding more tests or replacing a test with a better one, just wanted to know if I was missing something! :)